project/frontend/src/FindMatch/FindMatch.js

```diff
@@ -18,17 +29,24 @@ class PreferenceForm extends React.Component {
 
   // Updates the state of the input element so it holds the chosen value.
   changeState(event) {
```

Comment:
Planning to change this function to some clearer logic. Possibly could use useReducer for this

Reply:
will do in a follow up PR